lib/sourcemap.js

```diff
@@ -82,12 +82,22 @@ async function SourceMap(options) {
     }
 
     function add(source, gen_line, gen_col, orig_line, orig_col, name) {
+        let generatedPos = { line: gen_line, column: gen_col }
+
         if (orig_map) {
             var info = orig_map.originalPositionFor({
                 line: orig_line,
                 column: orig_col
             });
             if (info.source === null) {
+                if (generatedPos.column !== 0) {
```

Comment:
There's no need to check for this with the new source-map dependency. If this is a sourceless mapping and it's at generated column 0, it'll be discarded internally.